ports/openh264/vcpkg.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "openh264",
+  "version-string": "1.0.0",
```

```suggestion
  "version": "2.1.1",
```
This might be `version` here. Please see the doc https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions.